db/queries.py

```diff
@@ -3,44 +3,85 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    massages = list(Message.objects.filter(text__icontains=word))
+    return massages
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    untitled_chats = list(Chat.objects.filter(title__istartswith="untitled"))
+    return untitled_chats
 
 
 def get_users_who_sent_messages_in_2015() -> list[str]:
-    pass
+    queryset = list(
+        Message.objects.filter(sent__year=2015).values_list(
+            "user__first_name", "user__last_name"
+        )
+    )
+    return queryset
 
 
 def get_actual_chats() -> list[Chat]:
-    pass
+    queryset = list(Chat.objects.filter(message__sent__year__gt=2020))
+    return queryset
```

Comment:
that is not mistake, but you can make such functions in one line

Reply:
Yea i know, but i dont wanna change it. c:

Reply:
Heh 😯